examples/sec-sentiment-analysis/sec-sentiment-analysis.ipynb

```diff
@@ -0,0 +1,815 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "8534e135",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import tqdm\n",
+    "from fetch import get_form_by_ticker"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "820c49a1",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "tickers = ['ehc', 'mrk','nke', 'msex', 'v', 'cvs', 'doc', 'smtc', 'cl', \n",
+    "'ava', 'bc', 'f', 'lmt', 'cri', 'aig', 'rgld', 'apld', 'omcl', \n",
+    "'mmm', 'bgs', 'dis','wetg', 'bj']"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "f50b5e40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "cwd = os.getcwd()\n",
+    "data_directory = os.path.join(cwd, \"xbrl-forms\")\n",
+    "if not os.path.exists(data_directory):\n",
+    "    os.mkdir(data_directory)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b0b8e632",
+   "metadata": {},
+   "source": [
+    "If you're running this notebook at home, make sure to update the company and email as appropriate to correctly identify yourself to the SEC API."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "9b9d7c32",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "100%|██████████████████████████████████████████████████████████████████████| 23/23 [00:18<00:00,  1.23it/s]\n"
+     ]
+    }
+   ],
+   "source": [
+    "forms = []\n",
+    "for ticker in tqdm.tqdm(tickers): \n",
+    "    form_text = get_form_by_ticker(\n",
+    "        ticker=ticker,\n",
+    "        form_type=\"10-K\",\n",
+    "        company=\"Unstructured Technologies\",\n",
+    "        email=\"support@unstructured.io\"\n",
+    "    )\n",
+    "    \n",
+    "    filename = os.path.join(data_directory, f\"{ticker}-10k.xbrl\")\n",
+    "    with open(filename, \"w\") as f:\n",
+    "        f.write(form_text)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ce79d4e7",
+   "metadata": {},
+   "source": [
+    "### Extract the Risk Factors Narrative <a id=\"extract-narrative\"></a>\n",
+    "\n",
+    "Next, we'll extract the risk factors section by submitting the documents to the Unstructured SEC pipelines API. The SEC pipelines API accepts documents in XBRL format, finds the requested section, and returns the document as a JSON. You can learn more about the SEC pipelines API [here](https://github.com/Unstructured-IO/pipeline-sec-filings)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "d7727c8e",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import requests\n",
+    "import time"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "dc99e2be",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "url = \"https://api.unstructured.io/sec-filings/v0.1.0/section\""
```

Comment:
How do we keep this up to date?

Reply:
it could be parsed from https://github.com/Unstructured-IO/pipeline-sec-filings/blob/main/preprocessing-pipeline-family.yaml in the notebook.